src/server/rest/v1/service/TagService.ts

```diff
@@ -555,4 +636,24 @@ export default class TagService {
     );
     return result;
   }
+
+  private static async importTag(action: ServerAction, req: Request, tag: any): Promise<void> {
+    try {
+      const newUploadedTag: ImportedTag = {
+        id: tag.id.toUpperCase(),
+        description: tag.description ? tag.description : `Tag ID '${tag.id}'`,
```

Comment:
Use i18n?
I think we use the term 'Badge' in the UI.

Reply:
Done and changed also for AddDescriptionToTagsTask